installer/linux/installer/Readme.md

```diff
@@ -0,0 +1,2 @@
+# Protocol Handler for `ide://...` scheme.
```

Comment:
Lets be consistent - don't use mixed casing in files and use the same format. we have `README.adoc` in root, lets use same here.